autoflip_calculators/text_detection_calculator_test.cc

```diff
@@ -139,8 +141,8 @@ TEST(TextDetectionCalculatorTest, TwoPositionText) {
   // Setup test
```

Comment:
Add tests for cases were you expect it to return an error.  For example add a test if the model path is wrong.  Also, add a test for no text in the image.